helm/alfresco-content-services/7.0.N_values.yaml

```diff
@@ -2,30 +2,38 @@
 repository:
   image:
     tag: 7.0.1.10
-transformrouter:
+transformrouter: &ats-router
   image:
     tag: 1.4.1
-pdfrenderer:
+pdfrenderer: &ats-pdf
   image:
     tag: 2.5.7
-imagemagick:
+imagemagick: &ats-im
   image:
     tag: 2.5.7
-libreoffice:
+libreoffice: &ats-lo
   image:
     tag: 2.5.7
-tika:
+tika: &ats-tika
   image:
     tag: 2.5.7
-transformmisc:
+transformmisc: &ats-misc
   image:
     tag: 2.5.7
 aiTransformer:
   image:
     tag: 1.3.0
-filestore:
+filestore: &ats-fs
   image:
     tag: 0.16.1
+alfresco-transform-service:
+  transformrouter: *ats-router
+  pdfrenderer: *ats-pdf
+  imagemagick: *ats-im
+  libreoffice: *ats-lo
+  tika: *ats-tika
+  transformmisc: *ats-misc
+  filestore: *ats-fs
```

Comment:
I would put actual values here and move the previous section at the bottom of the file, with a comment which state they are there only for backward compatibility - so we can drop retrocompat easily later and users inspecting these files will not be confused by finding the former structure first

Reply:
you marked as resolved this but don't see any new commits

Reply:
oops thank to pre-commit push didn't happen.